### PR TITLE
Update docs for Pod Priority Based Node Graceful Shutdown beta

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@bobbypage ptal

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ different Kubernetes components.
 | `ExperimentalHostUserNamespaceDefaulting` | `false` | Beta | 1.5 | |
 | `GracefulNodeShutdown` | `false` | Alpha | 1.20 | 1.20 |
 | `GracefulNodeShutdown` | `true` | Beta | 1.21 | |
-| `GracefulNodeShutdownBasedOnPodPriority` | `false` | Alpha | 1.23 | |
+| `GracefulNodeShutdownBasedOnPodPriority` | `true` | Beta | 1.24 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:132]: This should be changed to 

```
| `GracefulNodeShutdownBasedOnPodPriority` | `false` | Alpha | 1.23 | 1.23 |
| `GracefulNodeShutdownBasedOnPodPriority` | `true` | Beta | 1.24 | |
```

**Comment by bradtopol** [comment]
Assigining @tengqm since he is already engaged on this.
/assign @tengqm

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (mrunalp) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:133]: fixed

**Comment by tengqm** [comment]
/lgtm
Deferring approval to another reviewer because I'm not sure if the feature is already merged upstream.

**Comment by mrunalp** [comment]
The feature is merged https://github.com/kubernetes/kubernetes/pull/107986

**Review by reylejano — APPROVED** [review]
Thank you for reviewing my previous comment and updating the PR
/approve

**Pull request merged**